src/units/abilities.cpp

```diff
@@ -1297,46 +1297,36 @@ static bool overwrite_special_affects(const config& special)
 	return (apply_to == "one_side" || apply_to == "both_sides");
 }
 
-unit_ability_list attack_type::overwrite_special_checking(const std::string& ability, unit_ability_list temp_list, const unit_ability_list& abil_list, const std::string& filter_self) const
+unit_ability_list attack_type::overwrite_special_checking(const std::string& ability, const unit_ability_list& temp_list, const unit_ability_list& abil_list, const std::string& filter_self, bool is_special) const
 {
 	bool overwrite_self = false;
 	bool overwrite_either = false;
-
+	unit_ability_list overwrite_list;
```

Comment:
As this is only used in the `else` block, it can be declared later in the function.

Reply:
Done